www/_template/guides/plugins.md

```diff
@@ -33,31 +33,57 @@ Create a directory for your plugin and a `.js` file using this template:
 // Example: a basic Snowpack plugin file, customize the name of the file and the value of the name in the object
 module.exports = function (snowpackConfig, pluginOptions) {
   return {
-    name: 'my-snowpack-plugin',
+    name: 'my-snowpack-plugin'
   };
 };
 ```
 
 To test your new plugin, run `npm init` to create a basic `package.json` then run `npm link` in your plugin’s directory to expose the plugin globally (on your development machine).
 
-For testing, [create a new, example Snowpack project](/tutorials/getting-started) in a different directory. In your example Snowpack project, run `npm install && npm link my-snowpack-plugin` (use the name from your plugin’s `package.json`).
+> Be aware that `npm install` will remove your linked plugin, so on any install, you will need to redo the `npm link my-snowpack-plugin`.
```

Comment:
Is this true? I thought `npm link` persisted across installs. But I haven’t used it too much so my knowledge might be wrong/outdated.

An alternative would be to maybe suggest `"my-snowpack-plugin": "file:../path/to/my-snowpack-plugin"` in deps instead? If it’s less finicky